raiden/tests/integration/api/test_restapi.py

```diff
@@ -346,7 +346,15 @@ def test_api_open_and_deposit_channel(api_server_test_instance, token_addresses,
         "settle_timeout": settle_timeout,
         "reveal_timeout": reveal_timeout,
     }
-
+    # First let's try to create channel with the null address and see error is handled
+    channel_data_obj["partner_address"] = "0x0000000000000000000000000000000000000000"
```

Comment:
it would be nicer IMO if constant NULL_ADDRESS_BYTES somehow was used here since it is testing correct handling of that case.

Reply:
That's bytes, here we are testing hex. We can have a null address hex constant but it would be equal to the string literal I used

Reply:
ok thank you for explanation. I noticed that in codebase one has to be very careful if he is working with bytes, hex or addresses. I am not involved enough to know where should what be used.

Reply:
Well I still agree with your comment though so I used a proper constant for the hexadecimal null address here and in the rest of the codebase